helpers/QtiPackageExporter.php

```diff
@@ -92,9 +96,14 @@ public function exportDeliveryQtiPackage(string $testUri): common_report_Report
      */
     public function exportQtiTestPackageToFile(string $testUri, string $fileSystemId, string $filePath): File
     {
-        $result = $this->exportDeliveryQtiPackage($testUri)->getData();
+        $pathToTempDir = $this->fileHelperService->createTempDir();
```

Comment:
Why removing the tmp dir here if we can remove on the method `exportDeliveryQtiPackage`? 

We also do not need the extra parameter `pathToTempDir`